actions/check-whl-on-pypi/action.yml

```diff
@@ -0,0 +1,32 @@
+---
+name: check whl on pypi
+description: check if specified whl already on pypi server or not
+
+inputs:
+  whl:
+    description: filepath of whl to check
+    required: true
+
+outputs:
+  status:
+    description: return true if whl not found, false if already existed
```

Comment:
This should be updated to reflect the actual values – `0` if no matching version is found, `1` if a matching version is found.

Reply:
I think `1`/`true` for exists and `0`/`false` for does not exist both make sense, as they do not follow the same logic as Linux (Unix?) status codes where `0` is success and everything else is failure, which tend to be unique and should not be used as an example for other uses of true/false or 0/1. For example, consider setting an environment variable that acts as a boolean flag – typically, `0` would mean off/disabled while `1` would mean on/enabled.

Regardless, the most important thing is that the description of the output value accurately describes the actual value returned and its significance. As long as that is corrected, it’s good.